remoulade/api/main.py

```diff
@@ -78,6 +80,27 @@ def requeue_message(message_id):
         return {"error": "requeue message in a pipeline not supported"}, 400
 
 
+@app.route("/messages/result/<message_id>")
+def get_results(message_id):
+    from ..message import get_encoder
+
+    max_size = 1e4
+    try:
+        backend = remoulade.get_broker().get_result_backend()
+        result = backend.get_result(message_id)
```

Comment:
i would use
`result = Result(message_id=message_id)`
It's the way to do on remoulade.